argutils/viz.py

```diff
@@ -58,6 +68,33 @@ def draw(ts, ax, use_ranked_times=None, tweak_x=None, arrows=False):
     )
 
 
+def label_nodes(ts, labels=None, metadata_name="name"):
```

Comment:
I can't see any reason why we'd change the name of the attr - why add all these extra parameters for a simple utility that we're only using in this paper? Let's keep it simple...

Reply:
Fair point. Are we happy with sticking with "name" then?